### ISSUE TITLE
Rename and enhance the "tatinFolder" parameter

### ISSUE THREAD

**Issue by aplteam** [aplteam/Cider#21]
1. "tatinFolder" binds Cider too closely to Tatin. It should be open for other dependencies as well
2. The syntax for telling principal dependencies from development dependencies is too complex.

For these reasons I suggest having this:

```
dependencies: {
    tatin: "packages",
    dado: "utilities",
},
dependencies_dev: {
    tatin: "packages_dev=TestCases",
},
```

**Comment by aplteam** [comment]
I am not sure about allowing the definition of more than one folder/namespace in a single entry like this:

```
dependencies_dev: {
    tatin: "packages_dev1=ClientTests,packages_dev2=ServerTests",
},
```

This is the old syntax. 

I tend to allow it because it does not hurt anything or anybody, although at the moment I cannot see an application for this.

**Comment by aplteam** [comment]
Implemented in 0.25.0
